sbe-tool/src/main/java/uk/co/real_logic/sbe/ir/generated/FrameCodecDecoder.java

```diff
@@ -15,6 +15,7 @@
     public static final int TEMPLATE_ID = 1;
     public static final int SCHEMA_ID = 1;
     public static final int SCHEMA_VERSION = 0;
+    public static final String SEMANTIC_VERSION = "null";
```

Comment:
This does not feel right. The string should be empty.

Reply:
Thank you for the review. It is fixed now.